src/styles/blocks/card.scss

```
@@ -0,0 +1,111 @@
+@import "../utiles/extends";
+@import "../utiles/resets";
+@import "../blocks/stars.scss";
+@import "../utiles/variables";
+
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 50px 40px;
+  margin: 0 auto;
+
+  &__container {
+    display: flex;
+    justify-content: space-between;
+    width: 944px;
+    align-items: center;
+    flex-wrap: wrap;
+  }
+}
+
+.card {
+  display: flex;
+  max-width: 200px;
+  flex-direction: column;
+  background-color: $white-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+  margin: 0 48px 46px 0;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__picture {
+    margin-top: 32px;
+    @extend %justify-center;
+
+    &-sizes {
+      @extend %justify-center;
+      width: 160px;
+      height: 134px;
+    }
+  }
```

`picture-sizes` it's a separate BEM element, not a modifier, you should't use this kind of nesting here. We use `-` to separate words in one class name.

more about naming in BEM in [video](https://mate.academy/learn/css-advanced/bem#/video/920)

```suggestion
  &__picture {
    margin-top: 32px;
    @extend %justify-center;
  }
  
  &__picture-sizes {
      @extend %justify-center;
      width: 160px;
      height: 134px;
    }
```